### DISCUSSION TITLE
GitHub without 2FA?!

### DISCUSSION THREAD

**Discussion by supervisitor** [community/community#66136] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

I am surprised, no one is questioning the introduction of 2FA in general? We are all in a discussion about the introduction of a mechanism for everyone, which actually only a small percentage, if any, requires.

### Body

Wouldn't it make more sense to discuss the rejection of the introduction of 2FA to all users instead of accepting the introduction and burning hours to implement it in a way that is acceptable to everyone? Remember, only a few really need it, implementation for everyone else is a waste of time. 😎
It's not that I want to declare 2FA as nonsense. But I have questions about the dictate that everyone has to use it. It seems to me like GitHub feel responsible when I accidentally run a red light! Why can we not still use the eMail code, if needed? Do they have concerns about the security of my email account?

Could we have a poll about 2FA? (yes, i need / no, I don't want / don't know)
Could we have 2FA as autonomously activatable option?
For those with security concerns, there should be a option per project:  "Code Editor only with 2FA login".
A "new issue" message should be always "open"/without 2FA...

A software version management system is easily installed by anyone who is serious here. Of course, GitHub also has advantages for collaboration in a global context. But without users and their projects, GitHub is as interesting as a private web business card. That's why I'd be interested to know why all too many accept the disadvantages of 2FA for themselves instead of getting upset about its forced introduction.

**Comment by howtophil** [comment]
2FA is "security theater" and not actually improved security. It does enhance the chances that you'll be forever locked out of your own account though.
